vita3k/interface.cpp

```diff
@@ -96,25 +96,39 @@ bool install_archive(HostState &host, GuiState *gui, const fs::path &path) {
     int num_files = mz_zip_reader_get_num_files(zip.get());
     fs::path sfo_path = "sce_sys/param.sfo";
     bool theme = false;
+    std::string extra_path = "";
 
     for (int i = 0; i < num_files; i++) {
         mz_zip_archive_file_stat file_stat;
-        if (!mz_zip_reader_file_stat(zip.get(), i, &file_stat)) {
+        if (!mz_zip_reader_file_stat(zip.get(), i, &file_stat))
             continue;
-        }
-        if (fs::path(file_stat.m_filename) == "sce_module/steroid.suprx") {
+
+        if (fs::path(file_stat.m_filename).string().find("sce_module/steroid.suprx") != std::string::npos) {
```

Comment:
can you just make fs::path(file_stat.m_filename).string() variable and use it below?